### PR TITLE
Newsletters: add 211 (2022-08-03)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic links, remaining PR summaries @harding 
- [x] Core Lightning 5441 @glozow (sorry it's not Bitcoin Core---there were no notable Bitcoin Core merges when I started writing)
- [x] Eclair 2253 @bitschmidty 

I might add an additional News section item depending on how on list discussion goes in the next couple days.

### CODE REVIEW

File: _posts/en/newsletters/2022-08-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Multiple derivation path descriptors:** Andrew Chow [posted][chow
+  desc] a [proposed BIP][bip-multipath-descs] to the Bitcoin-Dev mailing
+  list for allowing a single descriptor to specify two similar [BIP32][]
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-08-03-newsletter.md:16]: Perhaps:
```suggestion
  list for allowing a single descriptor to specify two related [BIP32][]
```

File: _posts/en/newsletters/2022-08-03-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    Replies from [Pavol Rusnak][rusnak desc] and [Craig Raw][raw desc]
+    indicated that Trezor Wallet and Sparrow Wallet already supported
+    the scheme Chow proposed.  Rusnak also asked whether a single
+    descriptor should be able to describe more than two similar paths.
+    Dmitry Petukhov [noted][petukhov desc] that only internal and
+    external paths are widely used today and that any additional paths
+    wouldn't have a clear meaning to existing wallets.  That could
+    create interoperability issues.  He suggested limiting the BIP to
+    just two paths and to wait for anyone needing additional paths to
```
Comment (adamjonas) [inline _posts/en/newsletters/2022-08-03-newsletter.md:44]: consistent with the gerund in the first half
```suggestion
    just two paths and waiting for anyone needing additional paths to
```

**Comment by harding** [comment]
Made suggested edits (thanks @Xekyo and @adamjonas ) and reviewed @bitschmidty's contribution (thanks!).  Added remaining content and topic entries.  Thanks all!

**Review by Xekyo — COMMENTED** [review]
LGTM

File: _posts/en/newsletters/2022-08-03-newsletter.md [hunk 3]
```diff
@@ -61,23 +56,39 @@ Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
 Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
 Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Core Lightning #5441][] hsmtool: implement checkhsm. FIXME:glozow
+- [Core Lightning #5441][] updates `hsmtool` to make it easier to check
+  a [BIP39][] passphrase against the [HD seed][topic bip32] used by
+  CLN's internal wallet.
```
Comment (glozow) [inline _posts/en/newsletters/2022-08-03-newsletter.md:61]: So sorry I missed this! Thanks :pray:

**Review by glozow — COMMENTED** [review]
LGTM

**Comment by bitschmidty** [comment]
Thanks @harding for authoring and @Xekyo and @adamjonas and @glozow for reviews! 🚀

**Pull request merged**